packages/cli/src/commands/integration/wizard.ts

```diff
@@ -207,11 +220,38 @@ async function getMetadataFromSteps(
 }
 
 function isHidden(schema: MetadataSchemaProperty) {
+  if (instanceOfExpression(schema['ui:hidden'])) {
+    throw new ExpressionError('Expression found in schema');
+  }
   return Boolean(
     schema['ui:hidden'] === true || schema['ui:hidden'] === 'create'
   );
 }
 
 function isReadOnly(schema: MetadataSchemaProperty) {
-  return Boolean(schema['ui:read-only']);
+  if (instanceOfExpression(schema['ui:read-only'])) {
+    throw new ExpressionError('Expression found in schema');
+  }
+  return Boolean(
+    schema['ui:read-only'] === true || schema['ui:read-only'] === 'create'
+  );
+}
+
+function isDisabled(schema: MetadataSchemaProperty) {
+  if (instanceOfExpression(schema['ui:disabled'])) {
+    throw new ExpressionError('Expression found in schema');
+  }
+  return Boolean(
+    schema['ui:disabled'] === true || schema['ui:disabled'] === 'create'
+  );
 }
+
+// biome-ignore lint/suspicious/noExplicitAny: <explanation>
+function instanceOfExpression(obj: any): obj is Expression {
```

Comment:
Nit: `unknown` instead of `any`?

Reply:
I always forget `unknown` is an option. Added.